api/src/services/initial-replication.js

```diff
@@ -0,0 +1,39 @@
+const db = require('../db');
+const authorization = require('./authorization');
+const purgedDocs = require('./purged-docs');
+const _ = require('lodash');
+const { DOC_IDS_WARN_LIMIT } = require('../services/replication-limit-log');
+
+const getContext = async (userCtx) => {
+  const info = await db.medic.info();
+  const authContext = await authorization.getAuthorizationContext(userCtx);
+  const docsByReplicationKey = await authorization.getDocsByReplicationKey(authContext);
+
+  const excludeTombstones = { includeTombstones: false };
+  const allowedIds = authorization.filterAllowedDocIds(authContext, docsByReplicationKey, excludeTombstones);
+  const unpurgedIds = await purgedDocs.getUnPurgedIds(userCtx.roles, allowedIds);
+
+  const excludeTombstonesAndTasks = { includeTombstones: false, includeTasks: false };
```

Comment:
does this mean we will not replicate tasks up during initial replication? is that a change in behaviour from today?